### PR TITLE
Addons implementation

### LINKED ISSUE #225: Addons implementation
### Expected behavior

We want to create different families of transformers and those to be optionally installed.

### PR DESCRIPTION
**Pull request by pvk-developer**
### Addons implementation

* New folder structure to contain the different addon families.
    * No `__init__.py` within this folder in order to dec
* New `identity` addon which contains:
    * `IdentityTransformer` : A simple test transformer which returns the input data.
    * Add integration tests for this transformer
* `addons_setup.py` : A generic `setup.py` file that creates distribution packages for `pypi`. **Bear in mind** this has to be run from the root of the project.
* Update to `Makefile` to run `addons_setup.py` when `make dist` is called (this creates a distribution for all the addons, however it can manually be triggered to only produce a single package by running `python rdt/transformers/addons/addons_setup.py <name of family>`.
* Dynamic import for installed addons within the `rdt/transformers/__init__.py` to allow the end user to import directly from `rdt.transformers`.

Resolve #225

### CODE REVIEW

**Review by fealho — APPROVED** [review]
Seems generally correct 👍

**Review by csala — APPROVED** [review]
LGTM! @amontanez24 adding you to the reviewers list to put this on your radar too

File: setup.cfg [hunk 1]
```diff
@@ -61,3 +61,4 @@ ignore-comments = yes
 ignore-docstrings = yes
 ignore-imports = yes
 disable = R0801, R0903, R0913, R0914, C0209, W0223, W0221, W0237
+
```
Comment (amontanez24) [inline setup.cfg:64]: Is this empty line supposed to be added?

**Review by amontanez24 — APPROVED** [review]
LGTM!

**Pull request merged**